### DISCUSSION TITLE
SAGE to include Sensor Insert

### DISCUSSION THREAD

**Discussion by shake73** [loopnlearn/loopbuildscripts#57] [Ideas]
Hi Marion,  Thank you for all that you do!  I wish I knew more about coding, but I am trying to learn.  I use NightGuard as a apple watch complication to monitor my son and it for some reason uses Sensor Insert (instead of Sensor Start).  So I was wondering if it would be possible to copy your patch for Sensor Start and simply change Start to Insert, and then include that in my customizations.  Currently I am inputting both manually into NS, but with your patch it automates the CAGE and SAGE which is awesome!
Thank you so much,
Erik Baker

**Comment by bjorkert** [comment]
What cgm are you using?

For G7 insert and start is the same thing. But for G6 users restarting their sensors and want to keep track of insert and (re)starts, having them separated could be valuable. I personally think it is a bit odd that NightGuard shows time since insert and not start, have you contact NightGuard in order to get a change there?
